analysis/museumdraw/analysis1_nov25_vssAbstract.Rmd

```diff
@@ -187,57 +244,48 @@ joint[,colInd]=scale(joint[,colInd])
-```{r}
-## Full model first
-fullModel=lmer(normCorrect ~ age + category + mean_intensity + num_strokes + draw_duration + (1|session_id), joint, REML = FALSE)
-
-# Check for normality assumptions on residuals
-s=summary(fullModel)
-hist(s$residuals)
-shapiro.test(residuals(fullModel))
-qqnorm(s$residuals)
-
-# Full model minus age
-noAge=lmer(normCorrect ~ category + mean_intensity + num_strokes+ draw_duration + (1| session_id), joint, REML = FALSE)
-
-## Model testing - model that includes age is better than model without age.
-modelOut=anova(fullModel,noAge)
-pOut=format.pval(modelOut$`Pr(>Chisq)`[2],1); # these quotes make it hard to embed in text...shorter variable for ease
 
-## Refit model for with REML for reporting betas
-## This does mean that the model beta is hard to interpret relative to the raw data but I don't see a good way around this
-fullModelReport=lmer(normCorrect ~ age + category + mean_intensity + num_strokes + draw_duration + (1|session_id), joint, REML = TRUE)
-
-coefOut=coef(summary(fullModelReport))
-ageBeta=format(round(coefOut[2,1],2),2)
-ageSE=format(round(coefOut[2,2],2),2)
-ageZ=format(round(coefOut[2,3],2),2)
+``` {r}
+mod <- glmer(correct ~ age + (1 | session_id) + (1 | category), 
+      data = joinedRatings,  
+      family = "binomial")
 
+kable(summary(mod)$coef, digits = 3)
 ```
+Now add some predictors, including  number of strokes, draw duration, and avg intensity (e.g., amount of ink). Standardize predictors for comparability. 
 
-## Alternative solution - generalized logistic mixed-effects models on raw rating data
-Get convergence warnings here even with minimal models -- ugh. -- commented out. If either of you see a better way to do these that I'm missing or have any suggestions I'd be happy to give these another shot.
 ``` {r}
-# joinedRatings$correctNum=as.numeric(joinedRatings$correct)
-# 
-# fullModel_glmer=glmer(correctNum ~ age + category + mean_intensity + num_strokes + draw_duration + (1+ age|session_id) + (1|category), data = joinedRatings,  family = "binomial")
-# 
-# reducedModel_glmer=glmer(correctNum ~ age + category + mean_intensity + (1|session_id), data = joinedRatings,  family = "binomial")
-# 
-# minimalModel_glmer=glmer(correctNum ~ age + category + (1|session_id), data = joinedRatings,  family = "binomial")
-
+mod_covariates <- glmer(correct ~ scale(age) + 
+                          scale(draw_duration) + 
+                          scale(mean_intensity) + 
+                          scale(num_strokes) +
+                        (1|session_id) + 
+                        (1 | category), 
+      data = joinedRatings,  
+      family = "binomial")
+
+kable(summary(mod_covariates)$coef, digits = 3)
 ```
 
+Poke around with interactions with age. Critical generalization is that the age increase still holds even controlling these factors, and there is no interaction. Could do this model a bunch of different ways, this is just one possibility. 
 
+``` {r}
```

Comment:
I scaled the variables and looked at two-way interactions as one possible way of looking at comparable coefficients in a simple-ish model, but this is quite exploratory.